lib/at_cmd_parser/at_utils.h

```diff
@@ -49,33 +50,6 @@ static inline bool is_notification(char chr)
 	return false;
 }
 
-/**
- * @brief Check if a string is a beginning of an AT command
- *
- * This function will check if the character is a "AT+" or "AT%" which
- * identifies an AT command.
```

Comment:
"AT#" as well :)

Reply:
I've already changed this description to `This function will check if the string is a valid AT command prefix.`